go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/stackrox/rox
 
-go 1.22.5
+go 1.23.2
```

Comment:
Can we do

```
go 1.23

toolchain 1.23.2
```

?

This way, we won't force anyone who uses this as a library to use 1.23.2, but rather any 1.23 version, while still forcing devs to use at least 1.23.2